src/components/Map.ts

```diff
@@ -184,14 +174,14 @@ export class Map extends Component<MapProps, MapState> {
 
     private createMakers(): Array<ReactElement<MarkerProps>> {
         const markerElements: Array<ReactElement<MarkerProps>> = [];
-        if (this.state.locations) {
+        if (this.state.locations && this.state.locations.length) {
             this.state.locations.map((locationObject, index) => {
                 const { latitude: lat, longitude: lng } = locationObject;

```

Comment:
if you don't use the short hand operator because you need as number, than we don't need to rename them in the destructing `const { latitude, longitude } = locationObject;`